sites/website/src/docs/resources/cheat-sheet.md

```diff
@@ -0,0 +1,695 @@
+---
+id: cheat-sheet
+title: Cheat Sheet
+sidebar_label: Cheat Sheet
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/resources/cheat-sheet.md
+---
+
+# Cheat Sheet
+
+## Packages:
+
+### [@microsoft/fast-element](https://www.fast.design/docs/fast-element/getting-started)
+
+**A library that is a lightweight means to easily build performant, memory-efficient, standards-compliant Web Components.**
+
+* Provides a thin layer of opinions on top of [Web Components](https://www.fast.design/docs/resources/why-web-components), lifting the level of abstraction just enough to make it easier and FASTer to [build components]((#building-components)).   
+* Use this library when you want to create new custom web components. 
+
+To install the fast-element library, use either `npm` or `yarn`:
+
+```shell
+npm install --save @microsoft/fast-element
+```
+
+```shell
+yarn add @microsoft/fast-element
+```
+
+Within your JavaScript or TypeScript code, you can then import library APIs like this:
+
+```ts
+import { FASTElement } from "@microsoft/fast-element";
+```
+
+---
+
+### [@microsoft/fast-foundation](https://www.fast.design/docs/introduction)
+
+**A library that provides foundational building blocks that can be assembled to create new design systems and component libraries.**
+
+* Exports of this package can generally be thought of as un-styled base components that implement semantic and accessible markup and behavior.   
+* Use this library when you want to implement something like Google's Material Design or Twitter Bootstrap.
```

Comment:
Is implement the right word here - perhaps "create" better convey's the intent? One other question on this is if Twitter is still associated with the latest versions of the project; I'm not sure they are. I did a quick search specifically for "Twitter Bootstrap" and it linked to _pinned_ versions of the project. The latest versions (4/5) don't seem to call out that relationship at all. I wonder if we can leave off that affiliation:

```suggestion
* Use this library when you want to create something like Google's Material Design or Bootstrap.
```

Reply:
@chrisdholt Thank you - updated!  I also updated the verbiage from the intro page: https://www.fast.design/docs/introduction

<img width="975" alt="Screen Shot 2021-12-14 at 2 26 04 PM" src="https://user-images.githubusercontent.com/35910788/146089266-8b031a4f-61d6-4f6b-ab9d-e49b7acf5c46.png">